### ISSUE TITLE
update to latest gridstack ?

### ISSUE THREAD

**Issue by adumesny** [pfms84/lb-gridstack#4]
the latest gridstack (0.6.3) - which has a lot of fixes - doesn't use lodash and a much smaller jquery-ui that is included in `gridstack.all.js` , so no need to include that much dependencies really.

Also would you be interested in adding Angular support in that lib instead ? (separate folder) if so ping me directly.

**Comment by stefanmatar** [comment]
This would be so much better. Also, could you upgrade the readme once this was done? It still says to include all those dependencies

**Comment by adumesny** [comment]
yeah I'm not sure why it's using 0.3, yet using latest Angular 9 (I'm still on 8 in my app).
GridStack 1.x has been out for a while now, and there will be  a native Typescript 2.x coming soon as well.

**Comment by adumesny** [comment]
gridstack 2.0 is out and native Typescript version now - still don't know why using latest Angular (well 10 is out) but VERY old gridtstack version - NO external dependency on latest gridstack and much smaller than 0.3
